samples/MQTTSN/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.10)
+
+PROJECT(azure-iot-udp-samples)
+
+add_subdirectory(${PROJECT_SOURCE_DIR}/lib/azure-sdk-for-c)
+add_subdirectory(${PROJECT_SOURCE_DIR}/lib/paho.mqtt-sn.embedded-c/MQTTSNPacket/src)
+add_subdirectory(${PROJECT_SOURCE_DIR}/lib/paho.mqtt-sn.embedded-c/MQTTSNPacket/samples)

```

Comment:
do we need this?

Reply:
I'm not sure how to reduce it more, here's what I currently have. 

`add_subdirectory(${PROJECT_SOURCE_DIR}/lib/azure-sdk-for-c)`

`add_subdirectory(${PROJECT_SOURCE_DIR}/lib/paho.mqtt-sn.embedded-c/MQTTSNPacket/src)`